hw09-jdbc/src/main/java/ru/otus/lantukh/jdbc/mapper/EntitySQLMetaDataImpl.java

```diff
@@ -0,0 +1,30 @@
+package ru.otus.lantukh.jdbc.mapper;
+
+public class EntitySQLMetaDataImpl<T> {
```

Comment:
Надо реализовать интерфейс.
```suggestion
public class EntitySQLMetaDataImpl<T> implements EntitySQLMetaData {
```